src/turnkeyml/build/stage.py

```diff
@@ -314,6 +319,10 @@ def launch(self, state: build.State) -> build.State:
 
         else:
             state.current_build_stage = None
+            state.build_status = build.Status.SUCCESSFUL_BUILD
+
+            state.results = copy.deepcopy(state.intermediate_results)
```

Comment:
Please add a comment on why this deepcopy is needed

Reply:
added in https://github.com/onnx/turnkeyml/pull/59/commits/a90de5b508494d775c079cd80e9a4009116ec58f